### ISSUE TITLE
Support for argument group with argparse

### ISSUE THREAD

**Issue by fury00812** [guildai/guildai#200]
It would be great to add support for `argument_group` as follows:

```python
parser = argparse.ArgumentParser()
group = parser.add_argument_group("Data")
parser.add_argument("—src", default="")
```

```yml
train:
  main: src/main
  flags-dest: args
    flags:
      src: "train.txt"
```

**Comment by gar1t** [comment]
Thanks for the report! This is on the road map for the next release.

**Comment by gar1t** [comment]
Sorry for the extremely late update, but this appears to be resolved, at least in 0.7.3 if not before. [This doc](https://github.com/guildai/issue-resolution/tree/master/200-support-for-argument-group-with-argparse) shows an example of the correct behavior.

Closing for now. If there are remaining issues, let's reopen.

**Comment by gar1t** [comment]
Dup of #89
